README.md

```diff
@@ -1,46 +1,29 @@
-# lichocker - [lichess.org](https://lichess.org) run in a [Docker](https://www.docker.com/) container
+# lichocker - [lichess.org](https://lichess.org) run as a [Docker](https://www.docker.com/) container stack
 
 ## Prerequisites
 
 * [Install Git](https://git-scm.com/downloads).
 * [Install Docker](https://docs.docker.com/install/).
+* [Install Docker Compose](https://docs.docker.com/compose/install/).
 * *Recommended*: Increase your Docker runtime memory to 4GB and your CPUs to 4 ([Mac](https://docs.docker.com/docker-for-mac/#advanced), [Windows](https://docs.docker.com/docker-for-windows/#advanced), and configure the `docker-machine` assigned memory and CPUs for VirtualBox).
 * Check out lichocker (`git clone https://github.com/BrandonE/lichocker`) and open the directory in your terminal: `cd /YOUR/PATH/TO/lichocker`
 * Check out [lila](https://github.com/ornicar/lila), the main project behind Lichess: `git clone --recursive https://github.com/ornicar/lila.git`
 * Check out [lila-ws](https://github.com/ornicar/lila-ws), which manages Websockets. `git clone https://github.com/ornicar/lila-ws.git`
 * Add the following line to your `/etc/hosts` file: `127.0.0.1 lichess-assets.local`
 
-## Obtaining the Docker image
-
-To obtain the image you can build it yourself or retrieve it from docker hub
-
-### Building the image
-
-Run the following while in the lichocker directory: `docker build --tag brandone211/lichess .`
+## Running
 
-### Retrieving from Docker Hub
+* lila and lila-ws source codes must be available to the containers. Hence, you must export their location:
 
 ```
-docker login
-docker pull brandone211/lichess
+export LILA_WS_REPOSITORY_DIR=/YOUR/ABSOLUTE/PATH/TO/lila-ws/
+export LILA_REPOSITORY_DIR=/YOUR/ABSOLUTE/PATH/TO/lila/
```

Comment:
Use a `.env` file instead, docker-compose can access it: https://docs.docker.com/compose/compose-file/#variable-substitution
default variable should look in the current directory.
Currently paths are lost if the shell is restarted.

Reply:
Good idea. I am also thinking at having lila and lila-ws as submodules of lichoker. That would make this redirection part way easier for any dev